cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -149,6 +150,7 @@ public class KafkaReconciler {
     private String logging = "";
     private String loggingHash = "";
     private final Map<Integer, String> brokerConfigurationHash = new HashMap<>();
+    private final Map<Integer, String> kafkaServerCertificateHash = new HashMap<>();
```

Comment:
is it on purpose, maybe related to KRaft naming things (kafka servers, controllers, ...), that we are moving from using `broker` prefix to `kafkaServer` here?

Reply:
yes, @tombentley suggested above that since we're introducing a new user facing API ,i.e. the annotation, we stay away from broker and use something more generic. So server was chosen.